app/models/info_request/sluggable.rb

```diff
@@ -0,0 +1,55 @@
+# A module to generate unique URL title slugs for Info Requests
+#
+module InfoRequest::Sluggable
+  extend ActiveSupport::Concern
+
+  included do
+    # make sure the url_title is unique but don't update
+    # existing requests unless the title is being changed
+    before_save :update_url_title,
+                if: proc { |request| request.title_changed? }
+  end
+
+  # When name is changed, also change the url name
+  def title=(title)
```

Comment:
I don't necessarily think this comment should block this merge, but I find it pretty uncomfortable that `#title=` is redefined here. Scanning through `InfoRequest`, this behaviour wouldn't be at all obvious. Its not an issue here, right now, but what would we do if some other concern also needed to perform a method on `#title=`?

I guess what would be nice is some way to run a callback on a per-method basis:

```ruby
after_receive :title=, -> { update_url_title }
```

Reply:
👌 nice!